src/status_im2/contexts/wallet/edit_account/view.cljs

```diff
@@ -9,15 +11,53 @@
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(defn show-toast
+  [{:keys [type theme]}]
+  (let [messsage (condp = type
```

Comment:
why `condp` and not just case? 🤔

Reply:
also typo -- `messsage` instead of `message` 🔍

Reply:
Yeah, `case` is a better choice here. 😅 

https://github.com/bbatsov/clojure-style-guide#case